server/sonar-web/src/main/js/store/users/reducer.js

```diff
@@ -20,31 +20,48 @@
 import { combineReducers } from 'redux';
 import uniq from 'lodash/uniq';
 import { RECEIVE_CURRENT_USER } from './actions';
+import { actions as membersActions } from '../../apps/organizations/store/membersDuck';
 
 const usersByLogin = (state = {}, action = {}) => {
-  if (action.type === RECEIVE_CURRENT_USER) {
-    return { ...state, [action.user.login]: action.user };
+  switch (action.type) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a "default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)